presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -137,6 +137,20 @@ public void testInnerInequalityJoinWithEquiJoinConjuncts()
     }
 
     @Test
+    public void testLeftConvertedToInnerInequalityJoinNoEquiJoinConjuncts()
+            throws Exception
+    {
+        assertPlan("SELECT 1 FROM orders o LEFT JOIN lineitem l ON o.orderkey < l.orderkey WHERE l.orderkey IS NOT NULL",
```

Comment:
I don't understand why LEFT JOIN is gone. Isn't ON condition part of LEFT JOIN? What does standard says?